examples/srd_sn/sn_srd_planck_bao.ini

```diff
@@ -0,0 +1,122 @@
+; Modeled closely to replicate the DES SN Cosmology results paper in
+; arXiv: 1811.02374
+
+[runtime]
+sampler = emcee
+root = ${PWD}
+;; For restarting a chain in a future run
+;; uncomment the following line
+;resume = T
+
+[default]
+fatal_errors = T
+planck_path = ${CSL_DIR}/likelihood/planck2018/baseline/plc_3.0
```

Comment:
`planck_path` is usually used to set paths to the Planck data, as specified elsewhere in the file. I don't see it used -- is that an oversight, or is the `planck_path` parameter not needed?